src/test-run/commands/assertion.js

```diff
@@ -55,8 +55,8 @@ export class AssertionCommand extends ActionCommandBase {
         ];
     }
 
-    static get NOT_REPORTED_PROPERTIES () {
-        return NOT_REPORTED_PROPERTIES;
+    _getNonReportedProperties () {
+        return super._getNonReportedProperties().concat(NOT_REPORTED_PROPERTIES);
```

Comment:
Let's rename the `NOT_REPORTED_PROPERTIES` to `ASSERTION_NOT_REPORTED_PROPERTIES`.